types/rxjs-exec/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for rxjs-exec 0.4
+// Project: https://github.com/kssfilo/rxjs-exec
+// Definitions by: Meno Abels <https://github.com/mabels>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as Rx from 'rxjs';
+
+export class Options {
+    binary?: boolean;
+    input?: string|Buffer;
+    stdin?: NodeJS.WritableStream;
+}
+
+export function exec(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+export function execFilter(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+export function mapExec(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+export function mapExecFilter(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+// declare export patch(rx: Rx.Observer);
```

Comment:
why is patch method missing ? https://github.com/kssfilo/rxjs-exec#install

Reply:
I'm not sure what you mean.  `require('rxjs-exec').patch(Rx.Observable);` is valid code and should be compilable right? If so, it would make sense to include it in the typing.

If `patch` doesn't belong here, then please remove the commented out code.

I can merge once you change hte code either way.